ResourceProvisioner/test/ResourceProvisioner.Infrastructure.UnitTests/Templates/NewProjectTemplateTests.cs

```diff
@@ -190,31 +190,46 @@ public async Task ShouldExtractBackendConfiguration()
             Acronym = workspaceAcronym
         };
 
-        var expectedConfiguration = @"resource_group_name = ""{{prefix}}-{{env}}-rg""
-storage_account_name = ""{{prefix_alphanumeric}}{{env}}{{suffix}}""
-container_name = ""{{prefix}}-project-states""
-key = ""{{prefix}}-ShouldExtractBackendConfiguration.tfstate""
-subscription_id = ""{{az_subscription_id}}""
-";
```

Comment:
@davidreneuw  I had to rewrite this test. It was comparint a string managed in a variable and a string managed in a file. In windows it would work proerly but would fail in linux. 

There was this really weird issue where windows/visual studio was including \r\n into the text comparison of the test that was looking for \n linux style line endings . 

It would be fine in Windows but when i updated the repo to respect/enforce the CRLF that we have set in our editorconfig the line style became an issue.